src/manual-edits/content/docs/new-relic-only/tech-writer-style-guide/processes-procedures/tech-writer-workflow.mdx

```diff
@@ -0,0 +1,134 @@
+
+### Developmental edit pass
+
+For cases where you have questions and suggestions rather than straight copy edits, follow these steps.
+
+1. Open the PR that you’re assigned to review.
+2. On the Files changed tab, you can either:
+    - Click Review changes and then select one of the following:
+      - **Comment** - use if you have a comment that doesn’t require follow up.
+      - **Approve** - use if you just want to approve the PR. You can request changes in the Leave a comment area, and select Approve if you want to let the writer make the edits and merge the file without a follow-up review from you.
+      - **Request changes** - use for times when you want to make sure the changes you request are included. You’ll be notified with any updates that the writer makes. 
+    - OR, start making comments on lines or sections of the doc. To do this, click the add comment icon , and leave an edit or comment for that specific line in the page. With this option, you get the choice between adding a single comment or starting a review. If you’re going to make comments throughout a doc, choose **Start a review** so the comments will all be rolled into one commit.
+3. Click **Finish your review** to complete your review. This triggers a notification to the writer alerting them that you’ve made suggestions. 
+
+### Copy edit workflow
+
+If you have copy edits for a file rather than comments and suggestions, you can make the changes to the file in different ways. Here are two main options:
+
+1. Edit using the GitHub browser: on the **Files changed** tab, in the diff window, click  and select **Edit file**. When you finish your edits, add a comment at the bottom of the file and choose to either **commit the changes directly**, or **create a branch and start a pull request**. Choose to branch and start a pull request if you expect a writer to review the diff and accept or revise your edits.
+2. Edit locally: check out the branch containing the file you want to edit, make the edits on your local drive, save, and commit your changes to the branch. 
+    - Note that this approach adds your edits to the open pull request. You can now see the changes you added to the file on the **Files changed** tab in the PR.
+
+These are just a few of many editing options. You’ll find your preferred way, just as you have with the Drupal tools.  
+
+### Revise and publish a doc
+
+If you’re notified that a reviewer has submitted a review to your file, go to your PR and review the changes. You might see them in the diff view, if they’re part of a review with comments; otherwise, they might appear as copy edits in the file. 
+
+1. Respond to any comments in the file. Either reply with follow up discussion, or click **Resolve conversation**.  
+1. When you’ve resolved all the comments, and all of the automatic checks have passed, you can merge the pull request. Merging the pull request sets in motion the automated build process and your changes will be published shortly. 
```

Comment:
should be a 2

Reply:
Also, won't the Docs team be handling these merges?

Reply:
@paperclypse - this doc is primarily for tech writers (mainly for members of our team).

Reply:
Good to know